### DISCUSSION TITLE
Minimum specs for 200 sites

### DISCUSSION THREAD

**Discussion by ghost** [OrchardCMS/OrchardCore#15435] [General]
What would be a good way to estimate how much cpu/ram/storage I would need if I wanted to run 200 or so sites using multi-tenancy?  Any pro tips would be greatly appreciated.

**Comment by hishamco** [comment]
I should ping @Piedone because they have a good SAAS solution called DotNest which of course has many websites :)

**Comment by Piedone** [comment]
This highly depends on what you do with those tenants, and whether the tenants run all the time (because you [can also shut them down when idle](https://github.com/Lombiq/Hosting-Tenants/tree/dev/Lombiq.Hosting.Tenants.IdleTenantManagement)). So, I suggest you do some testing for your specific use case, on the OS/platform you prefer for hosting, with a Release build.

To share some experiences, for us at [DotNest](https://dotnest.com/), running all kinds of ~250 OC tenants with idle shutdown works well under an Azure S1 Windows App Service (1 vCPU, 1,75 GB RAM).

**Reply by ghost** [reply]
Thanks for the info.  I just sent you a message through the contact form on your site.  We'd like a quote.

**Reply by Piedone** [reply]
Awesome! Will get back to you there.
